README.md

```diff
@@ -443,6 +451,44 @@ local sunglasses_options = {
 require("sunglasses").setup(sunglasses_options)
 ```
 
+### Config.can_shade_callback
+Default:  
+```lua
+-- lua
+function(opts)
+    -- opts: { window_id = number, buffer = number, filetype = string, filename = string }
+    return true
+end
+```
+
+This is a user-supplied callback function that runs when sunglasses
+determines if it can shade a window. Any return value other than true
+will result in said window not being shaded.
+
+An example of how to set this is as follows
+
+```lua
+    -- lua
+    local sunglasses_options = {
+        can_shade_callback = function(opts)
+            local conditions = {
```

Comment:
I like making this part of `sunglasses_defaults`

IE, by default users will have diffs "optionally" unshaded, though they can modify that logic if they choose. (by overriding this value in the provided default)